atoMEC/numerov.py

```diff
@@ -153,8 +155,10 @@ def KS_matsolve_parallel(T, B, v, xgrid):
                 -2 * xgrid
             )
 
-    # make temporary folder to store arrays
-    joblib_folder = "./joblib_memmap"
+    # make temporary folder with random name to store arrays
+    joblib_folder = "".join(
+        random.choices(string.ascii_uppercase + string.digits, k=30)
+    )
```

Comment:
Although very unlikely, but if a `joblib_folder` already exists it will still be (silently) overwritten. I would suggest 

```python
while True:
    try:
        joblib_folder = "".join(random.choices(string.ascii_uppercase + string.digits, k=30))
        os.mkdir(joblib_folder)
        break
    except FileExistsError as e:
        print(e)
```

Reply:
I've made the suggested change